File: Python Code/Daily_Flash_Code_0/Week 1/18Jan/05-DailyFlash_Solutions/05-DailyFlash_Solutions/18_Jan_Solutions_One/C/Evennumbers.c
/*
Program 4 : Write a Program to print First 50 Even Numbers
Output: 2 4 6 . . . 100
*/

#include<stdio.h>

#define lowerlimit 0
#define upperlimit 100

void main(){
	
	printf("First 50 even number\n");

	for(int itr = lowerlimit ; itr <= upperlimit ; itr++ ){
		
		if(itr%2==0)
			printf("%d ",itr);
	}	
		
}	

File: Python Code/Daily_Flash_Code_0/Week 1/18Jan/05-DailyFlash_Solutions/05-DailyFlash_Solutions/18_Jan_Solutions_One/JAVA/EvenNumbers.java
/*
Program 4 : Write a Program to print First 50 Even Numbers
Output: 2 4 6 . . . 100
*/

class EvenNumbers{

	public static void main(String args[]){

		System.out.println("First 50 even numbers");

		for(int itr = 0 ; itr <= 100 ; itr++ ){
		
			if(itr%2==0)
				System.out.printf("%d ",itr);
		}

	}	
}	
